### PR TITLE
Update data-classification-activity-explorer.md

### PR DESCRIPTION
**Pull request by shadansan**
@chrfox customers are asking why we have not documented the fact that Activity Explorer only reports on maximum 30 days worth of data. Hence adding it to the documentation

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/data-classification-activity-explorer.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: "Activity explorer rounds out the functionality of the data classif
...
+The [data classification overview](data-classification-overview.md) and [content explorer](data-classification-content-explorer.md) tabs give you visibility into what content has been discovered and labeled, and where that content is. Activity explorer rounds out this suite of functionality by allowing you to monitor what's being done with your labeled content. Activity explorer provides a historical view of activities on your labeled content. The activity information is collected from the Microsoft 365 unified audit logs, transformed and made available in the Activity explorer UI. Activity Explorer reports on up to 30 days worth of data.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer.md]:
```suggestion
The [data classification overview](data-classification-overview.md) and [content explorer](data-classification-content-explorer.md) tabs give you visibility into what content has been discovered and labeled, and where that content is. Activity explorer rounds out this suite of functionality by allowing you to monitor what's being done with your labeled content. Activity explorer provides a historical view of activities on your labeled content. The activity information is collected from the Microsoft 365 unified audit logs, transformed, and made available in the Activity explorer UI. Activity explorer reports on up to 30 days worth of data.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @shadansan, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**